### ISSUE TITLE
Accessibility Page Improvements

### ISSUE THREAD

**Issue by amandadupell** [grommet/hpe-design-system#1816]
### Is this issue related to the design system core library or design system website?

Design System website

### Small changes
- [ ] update Developer and Designer Guidance links (current ones are pointing to incorrect page)
- [ ] add external links to Software Design and Test Resources from [Product Accessibility Office](https://hpe.sharepoint.com/sites/f5/cto/office/Accessibility/Pages/accessible-web-software-and-hardware-design.aspx) to Resources section
- [ ] add government resources regarding laws surrounding accessibility (can be found under the [Governing Standards and Regulations](https://hpe.sharepoint.com/sites/f5/cto/office/Accessibility/Pages/accessible-web-software-and-hardware-design.aspx) section) to Resources section

### Component page-level changes
- [ ] for guides that outline accessibility guidance for certain components, include those direct links to the accessibility section for that component (i.e. this guide is specific to [creating tables](https://hpe.sharepoint.com/sites/F5/CTO/Office/Accessibility/Documents/Forms/AllItems.aspx?id=%2Fsites%2FF5%2FCTO%2FOffice%2FAccessibility%2FDocuments%2FCreating%20Accessible%20Tables%20in%20software%5Ffinal%5FMarch%202020%2Epdf&parent=%2Fsites%2FF5%2FCTO%2FOffice%2FAccessibility%2FDocuments&p=true&originalPath=aHR0cHM6Ly9ocGUuc2hhcmVwb2ludC5jb20vOmI6L3MvRjUvQ1RPL09mZmljZS9BY2Nlc3NpYmlsaXR5L0VheXJ6Y0Qxc3UxR3B0STQwSXVlWE9RQkc3WFRMZ05nb05LN1Y4RVJ0ZzdkLVE_cnRpbWU9QjM5RDhLaHMyVWc) and can be included on the Table component page)

### Section changes
- [ ] move Accessibility Guides from Resources bullet point list to its own section with direct links to the guide (this section maybe to include the three most recent guides written with some indication to highlight the newest one?) 
- [ ] move Accessibility Videos from Resources bullet point list to its own section with direct links to the video (this section could highlight general accessibility information like an [introduction to designing software for accessibility](https://web.microsoftstream.com/video/7f47a741-9bfe-40d0-b084-01f41fe12cc0), [accessible keyboard design](https://web.microsoftstream.com/video/7f47a741-9bfe-40d0-b084-01f41fe12cc0), etc. which could be helpful for onboarding) 

### Larger changes
- [ ] provide a form section where users can submit feedback regarding the content of the accessibility page, the videos and the guides; this would mostly be used by the Product Accessibility Office to know what kind of information people would like to see in the future 

### Open-ended Questions for Consideration
- How can we track the number of clicks we get on the guides and videos through this site?
- Should we consider breaking down the page into multiple pages based on the sections?
- How can we indicate an "onboarding" section or have a place to point to for new developers at HPE? 
- Should we have our own "checklist" for WCAG compliance, similar to [this one](https://webaim.org/standards/wcag/checklist) for WebAIM and others found [here](https://www.digitala11y.com/accessibility-cheat-sheets/)? 
- Most guides are written at a broad level without much technical language. Is there a way we could provide code snippets or examples through Grommet for developers?

**Comment by amandadupell** [comment]
### Additional Section
- [ ] add a "How to Best Test" section that outlines the best practices to adhere to when conducting accessibility audits and how to use assistive technology on your own to test products

This information could be pulled from some of the guides that Bill has written. Information is sometimes repeated in guides that we can abstract into this section. For example, when testing with a screen reader, your display monitor should be turned off to be an accurate experience of those relying solely on screen readers.

### Analytic Tracking
- [ ] confirm links to accessibility resources are tracked properly in Google Analytics

This data will be used by the Product Accessibility Office to see how much traffic we get from the Design System site. Helps us to answer the question of how valuable this page is to Design System users.
